### ISSUE TITLE
Error message with the pages which contain PDF or video files

### ISSUE THREAD

**Issue by unmick** [getgrav/grav-plugin-admin#2085]
Hello,

My current version is v1.7.5. Admin panel is v1.10.3

Since the 1.7x, for the pages with video or PDF files inside, 
![image](https://user-images.githubusercontent.com/35163557/109048015-d5fae280-76d6-11eb-97ef-2e9e8a954eba.png)

i get the following message when i open the page in the plug-admin

`TypeError:Cannot convert undefined or null to object
    at Function.keys ()
    at https://www.xxxxxxx.fr/user/sites/xxxxxxx.fr/plugins/admin/themes/grav/js/admin.min.js?88471a17ec:4150:16
    at https://www.xxxxxxx.fr/user/sites/xxxxxxx.fr/plugins/admin/themes/grav/js/admin.min.js?88471a17ec:1072:12
Call to a member function cropZoom() on null
`
The page media is empty. 
Hopefully i can still select my video as a media file of my page
![image](https://user-images.githubusercontent.com/35163557/109047347-0c842d80-76d6-11eb-99c5-1824f2520149.png)

When i select the document, it's ok for me the way it's displayed 
![image](https://user-images.githubusercontent.com/35163557/109047485-38071800-76d6-11eb-81cb-2ae0fffc74bb.png)

It would be nice if the system always shows all the files in the media zone and when it's not an image display an empty file .

**Comment by felixzwettler** [comment]
same issue, I have other files in the page directory which I want to link. The page still works, but gives me the error everytime I save it.

**Comment by mahagr** [comment]
I just uploaded a few files (jpg, pdf, mp4) into a test page and they all show up after saving without giving me an error. The only issue I can see is that after upload the thumbnail is missing from non-images.

Media only displays supported types, so if something is missing, you need to add the image extension and mime-type into media confguration.

What is the easiest way to reproduce the issue with a clean installation?

**Comment by unmick** [comment]
Hello,

I tried it in a test environment : it works.
I tried it in the main site in my live environment : it works.
I tried it once again in a multisite domain :  wrong.

in fact, i have added this in my setup.php in order to save the image of my theme under each site:
```
'image' => [
                'type' => 'Stream',
                'prefixes' => [
                    '' => ["user/{$folder}/image"],
                ]
            ] ,
```
I just remove it and it works for the thumbnail (Pdf, mp4....)  and , surprise, it's still properly working for the image of my theme that are recorded inside each site folder .

thanks for your help
